holoviews/ipython/__init__.py

```diff
@@ -152,6 +152,7 @@ def __call__(self, *args, **params):
             if p in self._backends:
                 imports.append((p, self._backends[p]))
         if not imports:
+            args = ['matplotlib']
```

Comment:
Was missing this out a bug? I assume this is what makes matplotlib the default if you do ``hv.notebook_extension()``.